src/gui/application.cpp

```diff
@@ -360,6 +354,14 @@ Application::Application(int &argc, char **argv)
 
     // Cleanup at Quit.
     connect(this, &QCoreApplication::aboutToQuit, this, &Application::slotCleanup);
+
+    QTimer::singleShot(0, this, [this] {
+        FolderMan::instance()->setupFolders();
+        _proxy.setupQtProxyFromConfig(); // folders have to be defined first, than we set up the Qt proxy.
+        for (const auto &ai : AccountManager::instance()->accounts()) {
+            slotAccountStateAdded(ai.data());
+        }
+    });
```

Comment:
Why are things initialized through the event loop now? 
Maybe worth a comment...

Reply:
Doing complex initialization in the constructor of qApp.....

Also this makes sure that qApp/ocApp is available one the folders get initialized.

Reply:
Maybe I should move that change in a separate review and also initialise the ui delayed.